lib/Widget/evaluationForms/previsitForm/healthStatusForm/healthStatus.dart

```diff
@@ -259,10 +257,9 @@ class _HealthStatusFormState extends State<HealthStatusForm> {
                                                                   signature) {
                                                                 if (!signature
                                                                     .hasData) {
-                                                                  return CircularProgressIndicator(
-                                                                    strokeWidth:
-                                                                        4,
-                                                                  );
+                                                                  return ProgressBar
+                                                                      .circularProgressIndicator(
+                                                                          context);
```

Comment:
Is it possible to change to CircularProgressIndicator reusable in other files.

Reply:
https://github.com/LaCare-KMUTT/AbdoCare-Web/pull/72/commits/bf0839f316524cc32040e4f64a906c9b2d45c2dd resolved here ja